sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -444,3 +449,69 @@ def display_data(self):
 
   def writer(self):
     raise NotImplementedError
+
+
+class PubSubSourceDescriptor(NamedTuple):
+  """A PubSub source descriptor for ``MultipleReadFromPubSub```"""
+  source: str
```

Comment:
It seems like the `source` here will be the full path to the topic/subscriptions. Please add more pydoc to describe what the source str should be.